music21/instrument.py

```diff
@@ -928,7 +975,6 @@ def __init__(self):
         self.instrumentSound = 'wind.reed.saxophone.baritone'
         self.midiProgram = 67
 
-        self.lowestNote = pitch.Pitch('B-3')
         self.transposition = interval.Interval('M-13')
```

Comment:
hey there Mark, chiming in to say I consider A3 to be the lowest note of the bari sax, which has definitely evolved over time into become more standard than nonstandard. More common IMO than the low B foot on a flute. What say you?

Reply:
Thanks @jacobtylerwalls . You're totally right that ranges change over time and will always be hard to pin down across the board. Probably the best approach for questions like this would be to take one (named) source as our guide deflect the questions on to them! In this case, most sources that I've seen still put written Bb as a standard lowest for all saxes.

Reply:
Cool. Out of scope for this PR I suppose.